guides/doc-Release_Notes/topics/katello-contributors.adoc

```diff
@@ -0,0 +1,15 @@
+Adam Růžička <adamruzicka@users.noreply.github.com>
+Bernhard Suttner <sbernhard@users.noreply.github.com>
```

Comment:
html check failed because of these email addresses. I suspect domains are wrong?

Reply:
Fixed.